packages/cli/src/applyPreset.ts

```diff
@@ -1,4 +1,13 @@
 // MUST import this file first before anything and not import any Lodestar code.
+
+// eslint-disable-next-line no-restricted-imports, import/no-extraneous-dependencies
+import {hasher} from "@chainsafe/persistent-merkle-tree/lib/hasher/as-sha256.js";
+// eslint-disable-next-line no-restricted-imports, import/no-extraneous-dependencies
+import {setHasher} from "@chainsafe/persistent-merkle-tree/lib/hasher/index.js";
+
+// without setting this first, persistent-merkle-tree will use noble instead
+setHasher(hasher);
```

Comment:
Does it make sense to set `as-sha256` as default in the library? Consumers still have the option to use noble but it is then their responsibility and we don't have to make sure it is set correctly in Lodestar.

Reply:
it's decided to use noble as default implementation, see https://github.com/ChainSafe/ssz/issues/313